source/designers-guide/getting-started/index.md

```diff
@@ -0,0 +1,257 @@
+---
```

Comment:
Shopware (the product) is always with capital S ("Shopware", not "shopware"). Please review the whole page for this typo